src/main/java/io/vertx/httpproxy/interceptors/HeadersInterceptor.java

```diff
@@ -12,29 +12,65 @@
 package io.vertx.httpproxy.interceptors;
 
 import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.codegen.annotations.Unstable;
 import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.MultiMap;
 import io.vertx.httpproxy.ProxyInterceptor;
+import io.vertx.httpproxy.interceptors.impl.HeadersFilterInterceptorImpl;
 import io.vertx.httpproxy.interceptors.impl.HeadersInterceptorImpl;
 
 import java.util.Set;
+import java.util.function.Consumer;
 
 import static io.vertx.codegen.annotations.GenIgnore.PERMITTED_TYPE;
 
 @VertxGen
+@Unstable
```

Comment:
In addition to the Javadoc for the methods, can you please also add Javadoc for the class?